Cargo.toml

```diff
@@ -11,4 +11,5 @@ license = "ISC"
 aargvark = "0.7"
 defer = "0.2"
 libc = "0.2"
+sd-notify = "0.4.5"
```

Comment:
Would you mind chopping this down to `0.4`? TBH I'm not really sure what the best practice is here, but for now I've been letting most deps float after their largest version number. Here I think any point changes should be fine (i.e. going to 0.4.6, etc) and this allows `cargo update` to update all the deps easily.